pkg/server/database.go

```diff
@@ -6,5 +6,5 @@ import "github.com/jhaals/yopass/pkg/yopass"
 type Database interface {
 	Get(key string) (yopass.Secret, error)
 	Put(key string, secret yopass.Secret) error
-	Delete(key string) error
+	Delete(key string) (bool, error)
```

Comment:
What is the bool suppose to signal? I'd say returning error is enough.

Reply:
The boolean indicated if the secret was deleted or not.
This allows the correct status code to be returned in [pkg/server/server.go line110-115](https://github.com/jhaals/yopass/pull/1026/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R110-R115).

I'm fine removing but to me it just seems nicer since we already require the check in [deploy/aws-lambda/main.go](https://github.com/jhaals/yopass/pull/1026/files#diff-47192465389a373f9270c2da55833733a851a8fea4dfcb5eb8002d167dab2870R76), [pkg/server/redis.go](https://github.com/jhaals/yopass/pull/1026/files#diff-318d9ffad7f2b955e423727fe62a40d43407c2ffb17ba4885dadb26fcacd9016R64) etc.